src/main/java/uk/gov/hmcts/reform/blobrouter/exceptionhandler/ResponseExceptionHandler.java

```diff
@@ -51,10 +52,9 @@ protected ErrorResponse handleInvalidSupplierStatementException(InvalidSupplierS
     }
 
     @ExceptionHandler(InvalidApiKeyException.class)
-    @ResponseStatus(HttpStatus.UNAUTHORIZED)
-    protected ErrorResponse handleInvalidApiKeyException(InvalidApiKeyException exception) {
+    protected ResponseEntity<String> handleInvalidApiKeyException(InvalidApiKeyException exception) {
         log.error(exception.getMessage(), exception);
-        return new ErrorResponse(exception.getMessage(), exception.getClass());
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(exception.getMessage());
```

Comment:
Now this is the only error that results in a different payload.

Reply:
what do you suggest? shall I leave it as it is now?

Reply:
Yeah, imho it's better to keep it consistent.
If we want to change it, let's change it for all errors.

Reply:
Also probably better to return json rather than just text